app/frontend/components/pages/staking/stakingPage.js

```diff
@@ -0,0 +1,103 @@
+const {h, Component} = require('preact')
+const connect = require('unistore/preact').connect
+const actions = require('../../../actions')
+
+class StakingPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      email: '',
+      emailValid: false,
+      errorMessage: '',
+    }
+
+    this.isValidEmail = this.isValidEmail.bind(this)
+    this.updateEmail = this.updateEmail.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+  isValidEmail(email) {
+    // eslint-disable-next-line max-len
+    const re = /^[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?$/gim
+    return re.test(email)
+  }
+
+  updateEmail(e) {
+    const isEmailValid = this.isValidEmail(e.target.value)
+    this.setState({
+      email: e.target.value,
+      emailValid: isEmailValid,
+      errorMessage: !isEmailValid && 'Invalid email format',
```

Comment:
```js
errorMessage: !isEmailValid && e.target.value !== '' && 'Invalid email format',
```
This way the email stays invalid in case its empty, but the error message does not get displayed.